src/client/examples/echo.rs

```diff
@@ -0,0 +1,57 @@
+use std::time::Duration;
+use fluvio::{FluvioError, Offset};
+use tracing_subscriber::EnvFilter;
+
+const TOPIC: &str = "echo";
+
+fn main() {
+    tracing_subscriber::fmt()
```

Comment:
use `fluvio_future::subscriber:init_logger`

Reply:
I don't plan on leaving the logger in the example